src/auto/pollable_input_stream.rs

```diff
@@ -0,0 +1,60 @@
+// This file was generated by gir (5a68ad0) from gir-files (469db10)
+// DO NOT EDIT
+
+use Cancellable;
+use Error;
+use InputStream;
+use ffi;
+use glib::object::IsA;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::mem;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct PollableInputStream(Object<ffi::GPollableInputStream, ffi::GPollableInputStreamInterface>): InputStream;
+
+    match fn {
+        get_type => || ffi::g_pollable_input_stream_get_type(),
+    }
+}
+
+pub trait PollableInputStreamExt {
+    fn can_poll(&self) -> bool;
+
+    //fn create_source<'a, P: Into<Option<&'a Cancellable>>>(&self, cancellable: P) -> /*Ignored*/Option<glib::Source>;
```

Comment:
Maybe this can be deignored

Reply:
Yes, just needs `Source` to be added to the toml file. glib has bindings for that